accounts/client_test.go

```diff
@@ -143,6 +144,160 @@ func TestCreateEntity(t *testing.T) {
 	}
 }
 
+func TestRetrievePaymentInstrumentDetails(t *testing.T) {
```

Comment:
Reorder this methods to be consistent with client order

Reply:
👍